tests/chocolatey-tests/commands/choco-outdated.Tests.ps1

```diff
@@ -4,6 +4,17 @@ Describe "choco outdated" -Tag Chocolatey, OutdatedCommand {
     BeforeAll {
         Initialize-ChocolateyTestInstall
         New-ChocolateyInstallSnapshot
+        # Pin all of the Chocolatey packages
+        @(
+            'chocolatey'
+            'chocolatey.extension'
+            'chocolatey-agent'
+            'chocolatey-management-database'
+            'chocolatey-management-service'
+            'chocolatey-management-web'
+        ) | ForEach-Object {
+            $null = Invoke-Choco pin add -n $_
+        }
         Invoke-Choco install upgradepackage --version 1.0.0 --confirm

```

Comment:
The pinning of the packages should be before calling `New-ChocolateyInstallSnapshot`, so we don't remove the pinning once we have more tests and call `Restore-ChocolaetyInstallSnapshot` (I included the Invoke-Choco call that was in the wrong place)

```suggestion
        # Pin all of the Chocolatey packages
        @(
            'chocolatey'
            'chocolatey.extension'
            'chocolatey-agent'
            'chocolatey-management-database'
            'chocolatey-management-service'
            'chocolatey-management-web'
        ) | ForEach-Object {
            $null = Invoke-Choco pin add -n $_
        }
        Invoke-Choco install upgradepackage --version 1.0.0 --confirm
        New-ChocolateyInstallSnapshot
```